core-bundle/tests/Controller/ContentElement/SwiperControllerTest.php

```diff
@@ -47,9 +47,15 @@ public function testOutputsMarkup(): void
             ],
         );
 
-        $expectedOutput = <<<'HTML'
+        $expectedJson = htmlspecialchars(json_encode([
+            'speed' => 300,
+            'offset' => 0,
+            'loop' => true,
+        ]));
+
+        $expectedOutput = <<<HTML
             <div class="content-swiper">
-                <div class="swiper" data-delay="0" data-speed="300" data-offset="0" data-loop>
+                <div class="swiper" data-settings="{$expectedJson}" data-delay="0" data-speed="300" data-offset="0" data-loop>
```

Comment:
I think we should remove the old data attributes from the template and just use `data-settings`. Customized versions of the template will continue to wok as we still set the old variables in the `SwiperController`. @fritzmg WDYT?

Reply:
Not if you only customized the Script block.

Reply:
True. But there is no BC promise for templates, so…